mzLib/Proteomics/Protein/Protein.cs

```diff
@@ -706,29 +678,28 @@ private void AddNterminalBiomarkers(int lengthOfProteolysis, int fullProteinOneB
         /// </summary>
         /// <param name="addFullProtein"> This needs to be added to the proteolysisProducts list to be searched </param>
         /// <param name="addForEachOrigninalProteolysisProduct"> the original products are there but those resulting from N- or C-terminal degradation still need to be added</param>
-        /// <param name="addNterminalDigestionBiomarkers"></param>
-        /// <param name="addCterminalDigestionBiomarkers"></param>
+        /// <param name="addNterminalDigestionTruncations"></param>
+        /// <param name="addCterminalDigestionTruncations"></param>
         /// <param name="initiatorMethionineBehavior"> this effects the intact proteoform as well as any original proteolysis products containing the N-terminus</param>
```

Comment:
initiatorMethionineBehavior is no longer a parameter

Reply:
good catch!

Reply:
fixed